### ISSUE TITLE
Add results for 'Other Fossil Fuels' if 0

### ISSUE THREAD

**Issue by MonikaFu** [RMI-PACTA/pacta.executive.summary#231]
From ES feedback, missed in the document maybe. Results for all exposure categories should be returned, even if they are 0.

**Comment by jacobvjk** [comment]
wait, I though we display it only if it is in at least one of the two asset_types. In that case it is fixed. If not, this is probably not just about other fossil fuels, but about each of the categories? @MonikaFu ?

**Comment by MonikaFu** [comment]
Yes, that is correct @jacobvjk. It is about all categories. Changed the title of this issue. Basically TMC want all 4 bars always to be present, even if there is no exposure to those (which makes sense to me honestly). So basically we need to make them 0 unless there are no results for an asset class at all, then we don't show results for that asset class.

**Comment by AlexAxthelm** [comment]
could this one be solved with `drop = FALSE` in `scale_x_discrete`?

>[drop](https://ggplot2.tidyverse.org/reference/scale_discrete.html)
Should unused factor levels be omitted from the scale? The default, TRUE, uses the levels that appear in the data; FALSE uses all the levels in the factor.

And it looks like we're converting to `factor` with all the levels as the last step of prepping the data.

**Comment by MonikaFu** [comment]
@jacobvjk exactly

**Comment by MonikaFu** [comment]
also - this is about the case when neither Equity nor Bonds have exposure to a certain category. Then we still want to show it in the plot.

**Comment by MonikaFu** [comment]
Because if one of them does, then it will show for the other automatically - because of the shared y axis.

**Comment by jdhoffa** [comment]
Just as an FYI, when we were on a call, this was flagged as a very low priority. I think if it's a hassle, just drop it

**Comment by MonikaFu** [comment]
I think that it is not a lot of hassle, just need to clearly explain to @jacobvjk what we need. Potentially can do it myself too but first need to do 2 other things.
